tests/integration/fuzzing/parameterized_test.cairo

```diff
@@ -0,0 +1,25 @@
+%lang starknet
+
+from starkware.cairo.common.math import assert_le
+
+@external
+func setup_examples() {
+    %{
+        max_examples(5)
+        # todo test no example
+        example(1, 2)
+        example(2, 3)
+        example(a=3, b=4)
+        example(b=6, a=5)
+        given(a = strategy.integers(15, 20), b = strategy.integers(10, 14))
```

Comment:
[Functionality] I am confused why is `given` cheatcode necessary here. It looks like a hypothesis requirement. I think Protostar should throw an exception if `example` is used without `given` telling the user that given is necessary.

Reply:
I think you don't have to use `given` here, from Protostar's perspective.

Under the hood we might want to detect usage of `example` without `given (this is trivial due to how TestConfig works), and apply a `given` decorator with only `explicit` phase enabled.

Reply:
This is correct, we do not have to use `given` cheatcode. Basically, hypothesis requires `given` when we use `example` but in protostar, `given` seems to be used automatically when not specified explicitly, so we're not going to get this error from hypothesis.